snaps_openstack/ansible_p/commission/openstack/playbooks/deploy_mode/kolla/multinode_kolla_compute.yaml

```diff
@@ -60,50 +59,48 @@
       git:
         repo: https://github.com/openstack/kolla-ansible.git
         version: "{{ GIT_BRANCH }}"
-        dest: "{{ GIT_CLONE_PATH_PIKE }}"
+        dest: "{{ GIT_CLONE_PATH_QUEENS }}"
```

Comment:
Seeing that we are only supporting a single version of OpenStack, we should consider renaming this variable something more generic like "OPENSTACK_PATH"

Reply:
after incorporating the changes related to this comment, we need to test the deployment completely, Request to please merge this PR and we can take this comment separately.